### PR TITLE
Blog Post: Maxwell Engineering in 2021 in numbers

### PR DESCRIPTION
**Pull request by stockandawe**

### CODE REVIEW

File: _posts/2021-12-31-2021-in-numbers.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Maxwell Engineering in 2021 in numbers
```
Comment (dpfeil) [inline _posts/2021-12-31-2021-in-numbers.md:3]: What about "by the numbers" so you don't have to repeat "in"?

File: _posts/2021-12-31-2021-in-numbers.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+author_avatar: https://avatars.githubusercontent.com/u/1410662?v=4
+author_name: Rutul Davé
+author_github_username: stockandawe
+---
+
+The end of calendar year is one of my favorite time of the year as I get to reflect and review the accomplishments and learnings of the past year. For the Maxwell Engineering Team, 2021 was an incredible year as we grew over 170%! Even more impressive though is how well every new contributor to our software stack exhibited [the Maxwell values](https://himaxwell.com/company/about-us) of rigor, ownership, curiosity, kindness, and being straight-up (#rocks) to make an impact towards our growth.
```
Comment (dpfeil) [inline _posts/2021-12-31-2021-in-numbers.md:9]: "one of my favorite times of the year"

Comment (dpfeil) [inline _posts/2021-12-31-2021-in-numbers.md:9]: "Even more impressive, though, is"

File: _posts/2021-12-31-2021-in-numbers.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+To get a flavor of that growth in numbers, I compiled some metrics around the engineering output in 2021.
+
+----
+
+## Lines of code
+
+At Maxwell, we don't rely on lines of code as a metric to quantify the output of engineering.
+LOCs are however fun to review at the end of a calendar year to reflect on the activities in some of our most active repositories.
```
Comment (dpfeil) [inline _posts/2021-12-31-2021-in-numbers.md:18]: "However, LOCs are fun to review"

File: _posts/2021-12-31-2021-in-numbers.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+![File changed, lines inserted, lines deleted](/images/posts/2021-12-30-2021-in-numbers/2021_Review_1.png)
```
Comment (dpfeil) [inline _posts/2021-12-31-2021-in-numbers.md:20]: `![Files changed, `

File: _posts/2021-12-31-2021-in-numbers.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+![Issues and epics completed](/images/posts/2021-12-30-2021-in-numbers/2021_Review_2.png)
+
+Over 200 epics completed this year includes all types of new features, enhancements, and even some re-architecting of critical services that power our web applications. Epics are broken down into units of stories and this year we closed out over 3,000 stories and over 1,000 individual bugs. This year we also invested in creating more tools and services to help our partners in Customer Support and Success help our customers more efficiently.
+
+To count some of these metics that most teams use for agile and project planning, I created a [small client using the GitHub API](https://github.com/stockandawe/gh_client) that we will be building out further and incorporate in our Sprint reviews and retrospectives in the future.
+
+----
+
+The numbers tell just a small part of the story of what we are able to accomplish as a team in a year. Each day I am grateful to get to work with some incredible smart and humble team members and build great software together.
```
Comment (dpfeil) [inline _posts/2021-12-31-2021-in-numbers.md:53]: "...incredibly smart.."

**Review by dpfeil — COMMENTED** [review]
Looking good! Just a couple suggestions. 🎉

File: _posts/2021-12-31-2021-in-numbers.md (hunk 2 above)
Comment (stockandawe) [reply _posts/2021-12-31-2021-in-numbers.md:9]: I am also confused by if it is "favorite time"or "favorite times" -- thanks!

Comment (dpfeil) [reply _posts/2021-12-31-2021-in-numbers.md]: If you say "one of" then it would be part of a set, so I think "favorite times" would make sense.

**Pull request merged**